app/presenters/queries/expanded_link_set.rb

```diff
@@ -64,8 +64,12 @@ def expanding_this_type?(type)
       end
 
       def dependees
-        link_set.links.group_by(&:link_type).each_with_object({}) do |(type, links), hash|
-          links = links.map(&:target_content_id)
+        return {} unless link_set.is_a? LinkSet
+        link_set.links
+          .pluck(:link_type, :target_content_id, :passthrough_hash)
+          .group_by(&:first)
+          .each_with_object({}) do |(type, links), hash|
+          links = links.map { |l| l[2] || l[1] }
```

Comment:
I thought that too, but apparently not. You're right though, it doesn't look good. I've pushed a change that makes it look nicer.